multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
```suggestion
    return x * y
```
I think this should be a multiply sign instead of an add sign